src/main/tray/tray-menu-item/tray-menu-item-injection-token.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectionToken } from "@ogre-tools/injectable";
+import type { IComputedValue } from "mobx";
+import type { LensMainExtension } from "../../../extensions/lens-main-extension";
+
+export interface TrayMenuItem {
+  id: string;
+  parentId: string;
+  orderNumber: number;
```

Comment:
Hmm. IMHO `orderNumber` as a name self-describes its behaviour. 1 is first, 2 is second and so on.

Reply:
It all depends on which direction the reader considers the "natural direction" since it could easily be "the higher the order number, the higher (more important) up the list it is (so closer to the top of the list)".